source/nodejs/adaptivecards/src/host-config.ts

```diff
@@ -241,10 +241,11 @@ export class RequiredInputLabelTextDefinition extends BaseTextDefinition {
 
 export class InputLabelConfig {
     inputSpacing: Enums.Spacing = Enums.Spacing.Small;
+    inputLabelWidth : number = 30;
```

Comment:
Should really be just `width` as `InputLabelConfig` already suggests this is configuration for "input labels". It's also consistent with the other setting names (i.e. `requiredInputs` and not `inputLabelsForRequiredInputs`)

Reply:
updated as `width`